### PR TITLE
Add global broadcast address

### PR DESCRIPTION
**Pull request by Liborsaf**

### CODE REVIEW

File: sonyapilib/device.py [hunk 1]
```diff
@@ -740,7 +743,9 @@ def send_authentication(self, pin):
 
     def wakeonlan(self, broadcast=None):
         """Start the device via wakeonlan."""
-        broadcast = broadcast or '255.255.255.255'
+        broadcast = broadcast or (self.broadcast_address
+                                  if hasattr(self, 'broadcast_address')
```
Comment (alexmohr) [inline sonyapilib/device.py]: the hasattr check isn't really necessary as it's always set in init?

Comment (Liborsaf) [reply sonyapilib/device.py]: I added this to prevent errors when loading old json file which doesn't have broadcast address stored yet

Comment (alexmohr) [reply sonyapilib/device.py]: Similar to the other MR, there should probably be a test case which creates a device from an 'old' and 'new' json

Comment (Liborsaf) [reply sonyapilib/device.py]: Will add test for broadcast address after merging the other MR so i will not add duplicate commits

Comment (Liborsaf) [reply sonyapilib/device.py]: Done

Comment (alexmohr) [reply sonyapilib/device.py]: Please let me resolve the threads next time. It is easier to review :)

**Comment by alexmohr** [comment]
When you're ready squash the commits please :)
